src/app/pages/virtualization/components/all-instances/instance-details/instance-disks/instance-disk-form/instance-disk-form.component.ts

```diff
@@ -63,6 +63,8 @@ export class InstanceDiskFormComponent implements OnInit {
     return !this.isNew() ? this.translate.instant('Edit Disk') : this.translate.instant('Add Disk');
   });
 
+  protected instance = computed(() => this.slideInRef.getData().instance);
```

Comment:
The computed callback doesn't have a signal call. This could just be a signal.
`protected instance = signal(this.slideInRef.getData().instance);` Or maybe doesn't need to be a signal